core/src/tasks/system/status/status.ts

```diff
@@ -95,9 +97,9 @@ export class StatusTask extends CLSTask {
   logSamples(samples: Status.StatusGetResponse) {
     if (env === "test") return;
 
-    const totalItems = [];
-    const pendingItems = [];
-    const pendingRuns = [];
+    const totalItems: Array<Record<string, number[]>> = [];
+    const pendingItems: Array<Record<string, number[]>> = [];
+    const pendingRuns: Array<Record<string, string[]>> = [];
```

Comment:
As per future code style guide
```suggestion
    const totalItems: Record<string, number[]>[] = [];
    const pendingItems: Record<string, number[]>[] = [];
    const pendingRuns: Record<string, string[]>[] = [];
```